### PR TITLE
Show new account immediately in account list

### LINKED ISSUE #591: New account does not immediately show up in account list
When I connect a new wallet to Alby. I see a Success! page. But the connection does not yet appear in the account list (top left). It only appears after I reload the page. 
I would expect that the connection immediately appears when I see the Success page!
image

### PR DESCRIPTION
**Pull request by dylancom**
Fixes #591

### CODE REVIEW

**Comment by dylancom** [comment]
I'm not so sure about the new "AccountInfo" type. Do you maybe have an idea here @bumi? It was confusing that we had an Account type in state and  a different Account type in our AuthContext.

**Comment by bumi** [comment]
@MoritzKa can you test this and confirm?

**Review by bumi — CHANGES_REQUESTED** [review]

**Comment by bumi** [comment]
yes, we have to be careful about the naming here, right? it kinda is confusing. AccountInfo holds mainly the response from  the getInfo calls whereas the Account describes the account itself?

**Comment by MoritzKa** [comment]
Worked well on Firefox. I couldn't test Chrome so easily, because somehow I cannot upload the zip packages from the Build files to  chrome://extensions/ anymore.

**Comment by bumi** [comment]
@MoritzKa thanks. do you have the "developer mode" enabled? (top right corner)

File: src/app/screens/Options/TestConnection/index.tsx [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
-        console.log(response);
-        const { alias } = response.info;
-        const balance = parseInt(response.balance.balance);
-        setAccountInfo({ alias, balance });
-      })
-      .catch((e) => {
-        console.log(e);
+    try {
+      const { currentAccountId } = await api.getStatus();
+      auth.setAccountId(currentAccountId);
```
[the remark this answers is not in the document]
Comment (dylancom) [reply src/app/screens/Options/TestConnection/index.tsx:31]: Well at this point accountId is already set in state.ts but our ContextProvider used in the front-end app is not aware of the latest accountid. If it fails then setAccountInfo won't we called within this screen and the balance / info won't we updated within the front-end app.

**Pull request merged**